builder/nutanix/config.go

```diff
@@ -262,3 +280,11 @@ func (c *Config) Prepare(raws ...interface{}) ([]string, error) {
 
 	return warnings, nil
 }
+
+// IsSHA256Checksum validates if the input string is a valid SHA-256 checksum
+func IsSHA256Checksum(input string) bool {
```

Comment:
Why are you checking this? The API checks for the checksum to be valid.

Reply:
goal is to check the format earlier in the packer config validation , to not wait a 100% sure error later

Reply:
Every time the API adds/removes supported algorithms, you would need to update your packer build though.
In my opinion it's better to leave this kind of error handling to the API.

Reply:
Or even in the golang sdk

Reply:
It's done immediately, and it's one API call, but sure. Then it should still be a part of the SDK, not anything implementing the SDK, because then you could make the argument that these checks need to be included in every tool using the API.

Reply:
clearly not immediately, but you are correct SDK would be the right place to do it

as i don't have a lot of time to pass on it right now, let's make a simple version first , without verification and we will improve later